app/views/site/_study_tabs_nav.html.erb

```diff
@@ -1,5 +1,5 @@
-<ul class="nav nav-tabs" role="tablist" id="study-tabs">
-  <li role="presentation" class="study-nav active" id="study-summary-nav"><a href="#study-summary" data-toggle="tab">Summary <i class="far fa-fw fa-file-alt"></i></a></li>
+<ul class="nav nav-tabs"  data-tablist-name="Study Overview" role="tablist" id="study-tabs">
```

Comment:
I remember @eweitz maybe said both words capitalized was the proper way... I forget why.

Reply:
My original comment compared "Study overview" and "Study Overview" while pairing.  The latter is preferred capitalization because it's the formal name of the page (appearing in the UI and Mixpanel in that form).

However, comparing between "Study Overview" and "study-overview", the latter seems even more preferable -- because it follows our pattern of logging specifically instrumented names in kebab-case.

So I'd opt for the tablist names in the format that Devon suggests, i.e.:
* Search -> search
* Study Overview -> study-overview
* Explore - Default -> explore-default
* Explore - Single gene -> explore-single-gene
* Explore - Multiple genes -> explore-multiple-genes

Reply:
9aded3d3